docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -114,11 +123,22 @@ in this template before creating a data stream. Later changes to the mappings or
 settings of a stream's backing indices may require reindexing. See
 <<data-streams-change-mappings-and-settings>>.
 
-The following <<indices-templates,put index template API>> request
-configures the `logs_data_stream` template.
+You can create an index template through the Kibana UI. From Kibana, open the
+menu and go to *Stack Management > Index Management*. In the *Index Templates*
+tab, click *Create template*. In the Create template wizard, use the *Data
+stream* toggle to indicate the template is used exclusively for data streams.
```

Comment:
```suggestion
You can create an index template through the Kibana UI:

. From Kibana, open the
menu and go to *Stack Management > Index Management*. 
. In the *Index Templates*
tab, click *Create template*.
. In the Create template wizard, use the *Data
stream* toggle to indicate the template is used exclusively for data streams.
```

Reply:
This might read better as an ordered list, though it would be different from the other Kibana directions that are inline.